src/components/sidebar/SafeList/index.tsx

```diff
@@ -68,119 +71,146 @@ const SafeList = ({ closeDrawer }: { closeDrawer: () => void }): ReactElement =>
     setOpen((prev) => ({ ...prev, [chainId]: open }))
   }
 
+  const isWelcomePage = router.pathname === AppRoutes.welcome
```

Comment:
Can we use the function from `SideDrawer` ? It is the same logic

Reply:
I removed this as per my other comment.